docs/source/essentials/server.md

```diff
@@ -104,7 +104,7 @@ And the server should return a simple response:
 }
 ```
 
-Your server itself is hosted at `http://localhost:4000/graphql`. This would be the endpoint you pass to set up Apollo Client.
+> The GraphQL server itself is also served at `http://localhost:4000/`, but only direct requests from a web browser will return the GraphQL Playground interface.  Therefore, Apollo Client will work properly when configured with this endpoint.
```

Comment:
I might change GraphQL server to `GraphQL data` or `GraphQL API`, since we want to move to a world where there is only Apollo Server that provides a GraphQL layer or GraphQL API